Backend/Lyzer/bin/Debug/net8.0/Lyzer.deps.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
@LittleClumsy could we look at removing all of the files in this directory and also adding it to the `.gitignore`?